Automattic/jetpack
# CRM: Start new 6.4.2-alpha cycle

## Proposed changes:

* This PR starts a new CRM 6.4.2-alpha cycle after the 6.4.1 release.

### Other information:


## Jetpack product discussion

p1709243149987899/1709237684.398989-slack-CTXBP902X


## Does this pull request change what data or activity we track or use?

n/a

## Testing instructions:

* Proof-read, check versions are correct (and readme version is the latest stable - 6.4.1).
* Possibly updates needed to changelog / readme - see p1709243149987899/1709237684.398989-slack-CTXBP902X
